package.json

```diff
@@ -20,7 +20,11 @@
   "main": "./dist/index.js",
   "module": "./dist/index.js",
   "types": "./dist/index.d.ts",
-  "dependencies": {},
+  "dependencies": {
+    "@types/storybook__addon-info": "^5.2.1",
```

Comment:
we do not need storyboook for the libarary user. put it to the devdependencies

Reply:
done